src/a-except.adb

```diff
@@ -161,556 +160,556 @@ package body Ada.Exceptions is
 
    procedure Rcheck_CE_Access_Check (File : System.Address;
                                      Line : Integer)
-      with Export,
-           Convention => C,
-           External_Name => "__gnat_rcheck_CE_Access_Check";
+     with Export,
```

Comment:
Could you please make the style of the aspect consistent with `platform.ads`?

```
procedure X
  with
    Export,
    Convention ...
```

Reply:
Fixed by https://github.com/Componolit/ada-runtime/pull/7/commits/f7181fb1083065b64927b930c745d5620c1d21d9.